ui/packages/docs/src/components/Frames/character_data.json

```diff
@@ -1110,5 +1110,19 @@
       "vid": "https://youtu.be/2VGB-csNGnI",
       "count": "https://docs.google.com/spreadsheets/d/1IQWPPH4rVSe9fvy2hpvufS1xi7Ni_jfZ7SL4aSudPuM/edit?usp=sharing"
     }
+  ],
+  "travelerhydro": [
+    {
+      "vid_credit": "Kolibri#7675",
+      "count_credit": "Kolibri#7675",
+      "vid": "https://youtu.be/MOyhu5YxeWA",
+      "count": "https://docs.google.com/spreadsheets/d/1_a0KTCdTkvITCoHvLzOzFbcRm1wMtDaHt7zFU9h2aPQ/edit?usp=sharing"
+    },
```

Comment:
Missing https://youtu.be/uwnKGID7vSI below this (copy everything and change vid).